docs/api/search/search-error.schema.json

```diff
@@ -0,0 +1,31 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "type": "object",
+    "title": "ErrorResult",
```

Comment:
`SearchErrorResult` would be a better name

Reply:
Done.